core/notifications/src/main/res/values/strings.xml

```diff
@@ -15,8 +15,8 @@
      limitations under the License.
 -->
 <resources>
-    <string name="sync_notification_title">Now in Android</string>
-    <string name="sync_notification_channel_name">Sync</string>
-    <string name="sync_notification_channel_description">Background tasks for Now in Android</string>
-
+    <string name="news_notification_title">Now in Android</string>
```

Comment:
The `news_` prefix feels redundant here for these 4 names

Reply:
The notifier package will house notifications for a other types too. The namespacing helps with this